cmd/kubeadm/app/master/manifests.go

```diff
@@ -57,30 +58,38 @@ const (
 // WriteStaticPodManifests builds manifest objects based on user provided configuration and then dumps it to disk
 // where kubelet will pick and schedule them.
 func WriteStaticPodManifests(s *kubeadmapi.KubeadmConfig) error {
+	// Placeholder for kube-apiserver pod spec command
+	apiServerCommand := getComponentCommand(apiServer, s)
+
+	// Check if the user decided to use an external etcd cluster
```

Comment:
`getComponentCommand` already has logic to append extra flags for different components based on flags, we should probably move this down there.